rollup.config.js

```diff
@@ -22,12 +22,16 @@ const external = Object.keys(globals);
 const babelOptions = () => {
   let result = {
     babelrc: false,
-    presets: [['env', { modules: false }], 'react'],
+    presets: [
+      ['@babel/preset-env', { modules: false }],
+      '@babel/preset-react',
+      '@babel/preset-flow',
+    ],
     plugins: [
       'emotion',
-      'transform-class-properties',
-      'transform-object-rest-spread',
-      'external-helpers',
+      '@babel/plugin-proposal-class-properties',
+      '@babel/plugin-proposal-object-rest-spread',
+      '@babel/plugin-external-helpers',
```

Comment:
@babel/plugin-external-helpers is not necessary
@babel/plugin-proposal-object-rest-spread is not necessary